backend/src/impl/benchmark_utils.py

```diff
@@ -278,6 +278,27 @@ def aggregate_view(
                         output_df = output_df.groupby(group_by).sum(numeric_only=True)
                     else:
                         output_df = output_df.sum(numeric_only=True)
+            elif op in {"add_default"}:
```

Comment:
This is named `add_default` but actually is adding a default over `ALL_LANG`, which is a bit more specific. I think we should either
1. Change the name of the `op` to something like `add_default_by_langauge` to make this a little bit more clear.
2. Add an option to `add_default` something like `"default_set": "all_lang"` to indicate that the set of things we should iterate over is coming from `ALL_LANG`. Then we could also add other options later.

I slightly prefer 2. if you have no preference.